### ISSUE TITLE
Teleport to [0,0,0] when using hoist hook

### ISSUE THREAD

**Issue by ampersand38** [Project-Hatchet/public_h-60#4]

**Comment by allegedflora** [comment]
Unfortunatly we just reproduced this on RC 0.3.1

https://youtu.be/bkBTKlxYEMg < Video

**Comment by BroBeansCPG** [comment]
Bug 1 -  If the winch is raised the player will be flung to a very distant mystical location

Working 1 - If the helicopter raises its altitude before the winch is raised (essentially manually taking the slack out of the winch line) and then the winch is raised the winch will operate properly.

Water 1 - No matter what we try this is the effect. 

[Videos/Media](https://drive.google.com/drive/folders/1rX8SDhzxbnla8f5Kon-htgz-hdg-QNQt)

**Comment by BroBeansCPG** [comment]
Maybe we need to explore using AttachTo rather than physically getting into the Hook

**Comment by ampersand38** [comment]
Hopefully closed by #199 for good

**Comment by andydvis** [comment]
@ampersand38 This is working far better than before so thanks for the work on it! I've been able to reproduce teleporting to [0,0,0] multiple times when the winch hook is too low to the ground and the player attaches to it, if the winch is above head height there is no issue.

Also reproduced being teleported to [0,0,0] and then back to original position in a loop when in the water and attached to the hook. If the helicopter raise then it starts to bring you up but I found that the rope then snaps.
